.changeset/three-kids-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": minor
+---
+
+`InputProps` interface changed to extend `InputHTMLAttributes<HTMLDivElement>` , `mergeA11yProps` removed, `ref` moved to container div
```

Comment:
`mergeA11yProps` was not exported before? so doesn't matter for consumers?

Could put a bit more wording on to the `ref` change, consequence of before / after

Reply:
updated